hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnection.java

```diff
@@ -242,7 +242,9 @@ public boolean write(OutboundFrame frame) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(acceptorSide, connectionId);
+        int result = (acceptorSide ? 1 : 0);
```

Comment:
Shouldn't this use the same value as `Boolean.hashCode`? Or even just calling that method?